src/System.Windows.Forms/src/System/Windows/Forms/DataGridViewRow.DataGridViewRowAccessibleObject.cs

```diff
@@ -35,7 +36,7 @@ private int VisibleIndex
                 ? dataGridView.ColumnHeadersVisible
                     ? dataGridView.Rows.GetVisibleIndex(_owningDataGridViewRow) + 1
                     : dataGridView.Rows.GetVisibleIndex(_owningDataGridViewRow)
-                : -1;
+            : -1;
```

Comment:
```suggestion
                : -1;
```
I guess this `:` should be aligned with `?` above not the `=>`